### PR TITLE
Address leftover feedback related to Skip/TrySkip APIs.

### PR DESCRIPTION
**Pull request by ahsonkhan**
Address the feedback that came up after the PR was merged: https://github.com/dotnet/corefx/pull/37793

### CODE REVIEW

File: src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs [hunk 1]
```diff
@@ -271,8 +271,21 @@ public bool Read()
         /// Skips the children of the current JSON token.
         /// </summary>
         /// <exception cref="InvalidOperationException">
-        /// Thrown when the reader was given partial data with more data to follow (i.e. _isFinalBlock is false).
+        /// Thrown when the reader was given partial data with more data to follow (i.e. IsFinalBlock is false).
```
Comment (bartonjs) [inline src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs]: Should that be `<see cref="IsFinalBlock"/>`?

**Comment by ahsonkhan** [comment]
@Anipik, @danmosemsft  - this merged without CI running (only two legs ran and they were green) because I added the auto-merge label. I think we should add some sort of delay at least to let Azure DevOps queue the other CI legs.

**Comment by Anipik** [comment]
I initially added a 5 minute check. I will increase the wait time.

**Comment by Anipik** [comment]
I just talked 5 minutes was the from the initial time of the PR. i am talking with them to add it after applying the label and as well if there is any new change to the pR.

**Pull request merged**